app/src/main/java/tr/edu/anadolu/bk/android/fragment/PageFragment.java

```diff
@@ -31,8 +38,34 @@ public void onCreate(Bundle savedInstanceState) {
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View view = inflater.inflate(R.layout.fragment_page, container, false);
-        TextView textView = (TextView) view;
-        textView.setText(String.format("Fragment #%s", page));
+
+        recyclerView = (RecyclerView) view.findViewById(R.id.recycler_view);
+        anAdapter = new AnnouncementAdapter(prepareAnnouncementData(), getContext());
+        recyclerView.setVisibility(View.INVISIBLE);
+        RecyclerView.LayoutManager mLayoutManager = new LinearLayoutManager(getContext());
+        recyclerView.setLayoutManager(mLayoutManager);
+        recyclerView.setItemAnimator(new DefaultItemAnimator());
+        if (page == 1) {
+            recyclerView.setAdapter(anAdapter);
+            recyclerView.setVisibility(View.VISIBLE);
+        }
+        prepareAnnouncementData();
```

Comment:
http://www.androidhive.info/2016/05/android-working-with-card-view-and-recycler-view/   kaynağı burdan aldım if  page adapteri ayarlamak için 1.sayfada  anAdapter kullanmak istiyordum sadece